File: cpp/1672-MaximumWealth.cpp
// 2022-4-14 1:38:55
// Runtime 4ms(89.0%)
// Memory 7.6mb(54.9%)

class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int max(0), current;
        for(int i(0); i < accounts.size(); i++) {
            current = 0;
            for(int j(0); j < accounts[i].size(); j++) 
                current += accounts[i][j];
            if(current > max)
                max = current;
        }
        return max;
    }
};

File: py3/1672-MaximumWealth.py
# 2022-4-14 1:30:40
# Runtime 36ms(70.1%)
# Memory 15mb(38.3%)

class Solution:
        def maximumWealth(self, accounts: List[List[int]]) -> int:
            ret = 0
            for client in accounts:
                if (cur := sum(client)) > ret:
                    ret = cur
            return ret
